src/main/java/levy/fallingsand/Main.java

```diff
@@ -5,6 +5,7 @@
     public class Main {
 
         public static void main(String[] args) {
+            new SandFrame().setVisible(true);
```

Comment:
Do not put this into your command line program

Reply:
what should I do instead

Reply:
What you were instructed to do...